### ISSUE TITLE
Pull broken (TypeError: projectNames.has is not a function)

### ISSUE THREAD

**Issue by icota** [dittowords/cli#23]
@azjgard https://github.com/dittowords/cli/commit/87feb8c2b09b8e6d2ec7bf54d06e2f9dd7fa2b7b#diff-d5109920b34c2b2e141bd41232d8876432a17479c388e142d8a036bb7b36ea62R86 introduced this bug:
```
ditto-cli pull
/usr/lib/node_modules/@dittowords/cli/lib/config.js:86
  if (projectNames.has(dedupedName)) {
                   ^

TypeError: projectNames.has is not a function
```

**Comment by azjgard** [comment]
Hi @icota, thanks so much for reporting this! 

We just published a resolution for this issue to NPM in version 2.1.2. Can you please pull down the latest release and confirm that the problem is now resolved for you?

**Comment by icota** [comment]
I can confirm this resolved in 2.1.2
Thanks for the swift fix @azjgard!
